src/Plaid/DateTimeConverter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Going.Plaid
```

Comment:
A namespace like _Going.Plaid.Json_ or _Going.Plaid.Json.Converters_ might be forward-looking in case there are more to create.

Reply:
Good call. I'll change namespace.